lib/Settings/Setting_wrapper.dart

```diff
@@ -0,0 +1,128 @@
+import 'package:cboard_mobile/Settings/ThemeManager.dart';
+import 'package:cboard_mobile/Settings/routes.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter/rendering.dart';
+import 'routes.dart';
+
+class Setting_wrapper extends StatelessWidget {
+  // ThemeNotifier theme = new ThemeNotifier();
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+      theme: ThemeData(
+        primaryColor: Color(0xFF23036A),
+        accentColor: Color(0xFFE3CAFF),
+        visualDensity: VisualDensity.adaptivePlatformDensity,
+        scaffoldBackgroundColor: Colors.white,
+      ),
+      home: Scaffold(
+        backgroundColor: Color(0xffE5E5E5),
+        appBar: AppBar(
+          centerTitle: true,
+          title: Text('Settings'),
+        ),
+        body: MyStatefulWidget(),
+      ),
+      routes: getRoute(),
+    );
```

Comment:
Here, I see that we're repetitively returning a `MaterialApp` with each iteration. Is there a way to return just a `Scaffold` to avoid a lot of the redundancy?

Reply:
Yes., sure. At first, I directly called this page so I put `MaterialApp` there.